Package.swift

```diff
@@ -18,6 +18,10 @@ let package = Package(
             from: "1.0.0"
         ),
         .package(
+            url: "https://github.com/apple/swift-argument-parser",
```

Comment:
Danger: This can cause Xcode to not even want to start building: https://stackoverflow.com/q/63079909/3939277

adding `.git` to the end of the URL will solve that, and also might fix the thing down below where you had to use `.product`

Reply:
Oh, a nice catch :)
Converted into an issue #24

Reply:
Glad to help 😁